coreneuron/io/global_vars.cpp

```diff
@@ -48,6 +48,7 @@ void set_globals(const char* path, bool cli_global_seed, int cli_global_seed_val
     (*n2v)["celsius"] = PSD(0, &celsius);
     (*n2v)["dt"] = PSD(0, &dt);
     (*n2v)["t"] = PSD(0, &t);
+    (*n2v)["PI"] = PSD(0, &pi);
```

Comment:
@iomaganaris : without looking into details, in the file mode everything work fine? i.e. PI is passed?

Reply:
Yes, I tried printing `pi` with both on and off line mode and it worked fine